app/services/users.js

```diff
@@ -25,7 +27,24 @@ const getUserByEmail = async email => {
   }
 };
 
+const getUsers = async (limit = DEFAULT_LIMIT, offset = DEFAULT_OFFSET) => {
+  try {
+    logger.info(`usersController::getUsers::offset::${offset}`);
+    logger.info(`usersController::getUsers::limit::${limit}`);
```

Comment:
This logs should be just one. I understand the convention, but the log add a lot of other information not needed and is better to have them together.